admin/sql/create_tables.sql

```diff
@@ -42,4 +42,45 @@ CREATE TABLE api_compat.session (
     ts        TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
 ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
+
+CREATE TABLE statistics.user (
+    user_id                 INTEGER NOT NULL, -- PK and FK to "user".id
+    artists                 JSONB,
```

Comment:
The plan is to use these fields to store the statistics we calculated from BigQuery. For example, in the artists field of `statistics.user`, we store the top artists listened to by that user over different time intervals. The json would be something like this, I guess: 

```
{
    "all_time": 
    [
        {
            "name": "example artist 1",
            "msid": uuid,
            "listen_count": 87
        },
        {
            "name": "example artist 2",
            "msid": uuid,
            "listen_count": 80
        }
    ],
    "last_month": 
    [
        {
            "name": "example artist 2",
            "msid": uuid,
            "listen_count": 12
        },
        {
            "name": "example artist 1",
            "msid": uuid,
            "listen_count": 3
        }
    ]
}
```

These stats would then be used to draw the graphs on the site. I don't think we will need to query specific values inside them ever, and we'll only write to them during our regular stat calculation timings, I guess.